slash-commands/slash-commands.js

```diff
@@ -67,15 +77,19 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 						var command = match[1];
 						var data = match[2];
 
+						// pass though some commands, they only merit formatting elsewhere
+						if ($.inArray(command, self.passthrough) != -1) { } 
 						// Match only whitelisted commands
-						if ($.inArray(command, self.commands) != -1) {
+						else if ($.inArray(command, self.commands) != -1) {
 							self[command](data);
-						} else {
-							// TODO: Better way to notify the user of the invalid command
-							alert("Invalid command: " + command);
+							args.message = '';
+						}
+						else {
```

Comment:
If you strip the space before sending it could be, but I don’t know any client having that behaviour.

You could implement it though, it could be what some users expect.